akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -61,11 +61,20 @@ object ActorMaterializer {
       system,
       materializerSettings,
       system.dispatchers,
-      context.actorOf(StreamSupervisor.props(materializerSettings, haveShutDown).withDispatcher(materializerSettings.dispatcher), StreamSupervisor.nextName()),
+      spawnStreamSupervisor(materializerSettings, context, haveShutDown),
```

Comment:
since this is in untyped world it might be better to use such name: `actorOfStreamSupervisor` ?

Reply:
Yup, okey :+